ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -133,7 +133,7 @@
   #set ($jobId = $services.drive.pack($paths, $packReference))
   #if ($jobId)
     #set ($discard = $downloadDoc.set('jobId', $jobId))
-    #if ($xwiki.checkAccess($downloadDoc, "edit"))
+    #if ($xwiki.hasAccessLevel("edit", $xcontext.getUser() , $downloadDoc))
```

Comment:
$services.security.authorization.hasAccess(...) has been introduced in 6.1RC1 according to http://extensions.xwiki.org/xwiki/bin/view/Extension/Security+Module#HScriptingAPI28since6.1RC129

File Manager needs to be compatible with XWiki versions starting with 5.4.4 -> https://github.com/xwiki-contrib/application-filemanager/blob/master/pom.xml#L30

I've sent this new commit https://github.com/xwiki-contrib/application-filemanager/pull/31/commits/ffb8a6945965dd3413c321193c52d1a9b36fbe6c which will hopefully please everyone :)